bin/create-release.sh

```diff
@@ -15,6 +15,27 @@ fi
 release_path="$1"
 release_name="$2"
 
+ensure_valid_dev_build_index() {
+	local dev_build_dir="$1"
+	local cache_dir="$2"
+
+	if [[ ! -e "${dev_build_dir}" ]]
+	then
+		return
+	fi
+
+	local indexes=$(find "${dev_build_dir}" -name index.yml)
+
+	for index in ${indexes}
+	do
+		local sha1=$(awk '/sha1/ {print $2}' < "${index}")
+		if [[ ! -e "${cache_dir}/${sha1}" ]]
```

Comment:
Do we have any of those for sha256 caches? I guess we would, if it was working at all…